great_expectations/data_context/store/data_context_store.py

```diff
@@ -41,6 +45,10 @@ def serialize(
         if self.ge_cloud_mode:
             assert isinstance(payload, dict)
             for attr in self.ge_cloud_exclude_field_names:
-                payload.pop(attr)
+                if attr in payload:
+                    payload.pop(attr)
+                    logger.debug(
+                        f"Removed {attr} from DataContextConfig while serializing to JSON"
+                    )
```

Comment:
Could also do it like this.
Not necessarily better, feel free to leave it as you have it.

```suggestion
                removed = payload.pop(attr, None)
                if removed:
                    logger.debug(
                        f"Removed {attr} from DataContextConfig while serializing to JSON"
                    )
```